pkg/parser/model/ddl.go

```diff
@@ -180,9 +181,97 @@ var actionMap = map[ActionType]string{
 	__DEPRECATED_ActionAlterTableAlterPartition: "alter partition",
 }
 
+// DDLBDRType is the type for DDL when BDR enable.
+type DDLBDRType string
+
+const (
+	// UnsafeDDL means the DDL can't be executed by user when cluster is Primary/Secondary.
+	UnsafeDDL DDLBDRType = "unsafe DDL"
+	// SafeDDL means the DDL can be executed by user when cluster is Primary.
+	SafeDDL DDLBDRType = "safe DDL"
+	// UnmanagementDDL means the DDL can't be synced by CDC.
+	UnmanagementDDL DDLBDRType = "unmanagement DDL"
+	// UnknownDDL means the DDL is unknown.
+	UnknownDDL DDLBDRType = "unknown DDL"
+)
+
+// ActionBDRMap is the map of DDL ActionType to DDLBDRType.
+var ActionBDRMap = map[ActionType]DDLBDRType{
```

Comment:
Do you think it's better to group the them by their DDLBDRType?

Reply:
I think your concern is that `group the them by their DDLBDRType` can more visually see its classification.

But that `group by ActionType` is more easily used in code.

So, I can add another Map to show its classification, and add a new ut to keep them consistent.

Reply:
Maybe we can initialize `ActionBDRMap` in `init()` like https://github.com/pingcap/tidb/blob/7ef4b4bb69a2e0d90ee19c6b23c7be8ec8ce2e0d/pkg/expression/builtin_convert_charset.go#L300

Thus, the UT is not necessary.